src/Gateway.php

```diff
@@ -1045,9 +1045,14 @@ private function set_base_payment_data($payment, $order)
         $payment->setStamp( get_current_blog_id() . '-' . $order->get_id() . '-' . time() );
 
         // Create a random reference for the order
-        $reference = sha1( uniqid( true ) );
-
         // Create a unique hash to be used as the request reference
+        //$reference = sha1( uniqid( true ) );
```

Comment:
Not sure do we need to keep these old things and trello references in history. What you think @loueranta ?

Reply:
No need to keep these, please remove them.

Reply:
Comments removed, pull request updatred.